src/sentry/static/sentry/app/views/projectChooser.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {History} from 'react-router';
+
+import OrganizationState from '../mixins/organizationState';
+import TodoList from '../components/todos';
+
+const ProjectChooser = React.createClass({
+  mixins: [
+    OrganizationState,
+    History
+  ],
+
+  componentWillMount() {
+    jQuery(document.body).addClass('narrow');
+  },
+
+  componentWillUnmount() {
+    jQuery(document.body).removeClass('narrow');
+  },
+
+  render() {
+    let org = this.getOrganization();
+
+    // Expect onboarding=1 and task=<task id> parameters and task.feature_location == 'project'
+    // TODO throw up report dialog if not true
+    let task = TodoList.TASKS.filter((t) => t.task == this.props.location.query.task)[0];
+    if (task.feature_location != 'project') {
+      throw new Error('User arrived on project chooser without a valid task id.');
+    }
+    let teamProjectList = [];
+
+    org.teams.map((team) => {
+      let projectList = [];
+      team.projects.map((project) => {
```

Comment:
.forEach returns undefined. I could modify the array in place, but using .map feels cleaner.